src/coreclr/tools/aot/ILCompiler.Reflection.ReadyToRun/TransitionBlock.cs

```diff
@@ -169,5 +172,21 @@ private sealed class LoongArch64TransitionBlock : TransitionBlock
             public override int OffsetOfFirstGCRefMapSlot => SizeOfCalleeSavedRegisters;
             public override int OffsetOfArgumentRegisters => OffsetOfFirstGCRefMapSlot;
         }
+
+        private sealed class RiscV64TransitionBlock : TransitionBlock
+        {
+            public static readonly TransitionBlock Instance = new RiscV64TransitionBlock();
+
+            public override int PointerSize => 8;
+            // a0 .. a7
+            public override int NumArgumentRegisters => 8;
+            // fp=x8, ra=x1, s1-s11(R9,R18-R27), tp=x3, gp=x4
+            public override int NumCalleeSavedRegisters => 15;
```

Comment:
In the comment, total fp, ra, s1-s11, tp and gp is 14. Could you check which is correct?

Reply:
It seems that both are 15, or am I missing smth? x1, x3, x4, x8, x9, x18, x19, x20, x21, x22, x23, x24, x25, x26, x27

Reply:
You are right! Thank you!